### DISCUSSION TITLE
stitching multiple images

### DISCUSSION THREAD

**Discussion by jetsonwork** [OpenStitching/stitching#136] [Q&A]
Hi,
I have around 1000 images and I want stitch them, since they are in high resolution and I have to do some processing afterward I don't want to resize them. could you please let me know what is the best approach to stitch these images? I stitch images batch by batch (50 images in each batch), but after making the batches I could not stitched these batches together.

**Comment by lukasalexanderweber** [comment]
So e.g. two batches are working independently but stitching the two subresults fail?

**Reply by jetsonwork** [reply]
Thank you, Lukas, for your prompt response. Let me elaborate further. For instance, I have 200 images. I can successfully stitch them in batches of 2x100 images (meaning, after this step, I have two stitched batches, each containing 100 images). However, when I try to stitch these two batches together to create the final panorama, it fails.

**Reply by lukasalexanderweber** [reply]
Could you share sample Images?

**Reply by jetsonwork** [reply]
In the link provided below, you will find one sample image. These images were taken by a drone over an agricultural field. If we consider the field as a rectangle, the images were taken along the length of the rectangle. (sorry for some limitation I cannot share more images) 

https://imageupload.io/yvjQqnhIbNpuXMq

**Reply by lukasalexanderweber** [reply]
You might need to use `AffineStitcher` for such images. See e.g. https://github.com/OpenStitching/stitching/discussions/135

**Reply by jetsonwork** [reply]
Thank you. When I'm creating the batches (not the final image) and also the final image, what settings do you recommend for my dataset? I'm currently using the settings below. Additionally, does the order of the images matter? And can I use the --no-crop option? I'm concerned that if I use cropping, I might lose some parts of the images.


stitch * --confidence_threshold 0.2 --wave_correct_kind auto --detector akaze

**Reply by lukasalexanderweber** [reply]
Maybe try

stitch * --confidence_threshold 0.2 --affine --detector akaze

**Reply by jetsonwork** [reply]
Thanks you. could you please let me know whether it is possible to use the GPU or not to speed up the process?

**Reply by lukasalexanderweber** [reply]
No you cant https://github.com/OpenStitching/stitching/discussions/90

**Reply by jetsonwork** [reply]
Thank you so much for your prompt responses. In my case, would it be better to use the 'crop' or 'no-crop' option for the batches I'm creating? If I choose 'no-crop', will the algorithm still be able to stitch images that have black borders?

**Reply by jetsonwork** [reply]
however sometime I am getting below error

 ** On entry to DLASCLS parameter number  4 had an illegal value

**Reply by lukasalexanderweber** [reply]
Just try it out. At this point, I wont get any further into support. Consider sponsoring this repo if you need more help.
